### PR TITLE
Added api guide for check-in

### PR DESCRIPTION
**Pull request by khenrichsen**

### CODE REVIEW

**Review by rebekaburnett — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -10,11 +10,9 @@ END_METADATA -->
...
 # Vipps Check in API
 
-The Check in API is not available for everyone yet.
-See the
-[OpenAPI Spec](https://vippsas.github.io/vipps-check-in-api/redoc.html)
-for a sneak peek.
-
+The Check in API is in beta, and is available to use for merchants that work with [Loyalty in POS](https://vippsas.github.io/vipps-developer-docs/docs/vipps-solutions/loyalty-in-pos/)
+* [API Guide](vipps-check-in-api.md): Developer guide for the Vipps QR API.
+* [API Reference](https://vippsas.github.io/vipps-developer-docs/api/check-in): QR API Reference Specifications.
```
Comment (rebekaburnett) [inline README.md:15]:
```suggestion
* [API Reference](https://vippsas.github.io/vipps-developer-docs/api/check-in): QR API Reference Specifications.

```

File: vipps-check-in-api-how-it-works.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <!-- START_METADATA
 ---
-draft: true
+title: Checklist
+sidebar_position: 40
```
Comment (rebekaburnett) [inline vipps-check-in-api-how-it-works.md]:
```suggestion
sidebar_position: 10
```

File: docs/index.html [hunk 3]
```diff
@@ -48,7 +48,7 @@
 window.onload = function() {
   // Build a system,
   const ui = SwaggerUIBundle({
-    url: 'https://vippsas.github.io/vipps-check-in-api/swagger.yaml',
+    url: 'https://vippsas.github.io/vipps-qr-api/swagger.yaml',
```
Comment (Sigvedash) [inline docs/index.html]: vipps-qr-api ??

Comment (khenrichsen) [reply docs/index.html]: fixed

**Pull request merged**